src/AttributesAndMethods.jl

```diff
@@ -71,19 +103,35 @@ struct map_from_weil_divisors_to_class_group
 end
 export map_from_weil_divisors_to_class_group
 
+
+"""
+    map_from_weil_divisors_to_class_group( v::NormalToricVariety )
+
+Computes the map from the Weil divisors to the Class group of the normal toric variety `v`.
+"""
 function map_from_weil_divisors_to_class_group( v::NormalToricVariety )
     gap_MapFromWeilDivisorsToClassGroup = GAP.Globals.MapFromWeilDivisorsToClassGroup( v.GapNTV )
     return map_from_weil_divisors_to_class_group( gap_MapFromWeilDivisorsToClassGroup )
 end
 export map_from_weil_divisors_to_class_group
 
 
-function dimension( v::NormalToricVariety )
+"""
+    dim( v::NormalToricVariety )
+
+Computes the dimension of the normal toric variety `v`.
+"""
+function dim( v::NormalToricVariety )
     return GAP.Globals.Dimension(v.GapNTV)::Int
 end
-export dimension
+export dim
+
 
+"""
+    dimension_of_torusfactor( v::NormalToricVariety )
```

Comment:
Should we change this to `dim_of_torusfactor` analogously then?

Reply:
Yes, that is a good point. Changed as well.